ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -133,11 +133,12 @@ message DamlLogEntryId {
 }
 
 message DamlSubmitterInfo {
-  string submitter = 1;
+  repeated string act_as = 1;
   string command_id = 2;
   string application_id = 3;
   reserved 4; // was maximum_record_time
   google.protobuf.Timestamp deduplicate_until = 5;
+  repeated string read_as = 6;
```

Comment:
We an remove this as it won't be necessary if we drop the visibility check on the committer.